infrastructure/code/code_html.go

```diff
@@ -46,6 +46,37 @@ func getDataFlowHeadingHtml(issue snyk.CodeIssueData) string {
 	return fmt.Sprintf("Data Flow - %d %s", dataFlowCount, stepWord)
 }
 
+func getIgnoreDetailsHtml(ignoreDetails snyk.IgnoreDetails) string {
+	ignoreDetailsHtml := fmt.Sprintf(`<div class="ignore-details-column">
+  %s
+	%s
+</div>
+<div class="ignore-details-column">
+  %s
+	%s
+</div>
+%s
+`,
+		getIgnoreDetailsRow("Category", ignoreDetails.Category),
+		getIgnoreDetailsRow("Expiration", ignoreDetails.Expiration),
+		getIgnoreDetailsRow("Ignored On", ignoreDetails.IgnoredOn.String()),
+		getIgnoreDetailsRow("Ignored By", ignoreDetails.IgnoredBy),
+		getIgnoreDetailsRow("Reason", ignoreDetails.Reason),
+	)
+	warning := `Ignores are currently managed in the Snyk web app.
+To edit or remove the ignore please go to: <a href="https://app.snyk.io" target="_blank" rel="noopener noreferrer" >https://app.snyk.
+io</a>` // TODO: what about different env
```

Comment:
I wanted to compute the different URL here in case someone is running it in MT-EU or whatever, but I see this isn't done anywhere else: https://github.com/snyk/snyk-ls/blob/main/infrastructure/oss/issue_html.go#L82

Am I going too far or should I raise a ticket to address both these places in a follow-up PR?

Reply:
We could add this check it IDE-244 🙌